test/tests/functional/pbs_basil_support.py

```diff
@@ -38,8 +38,7 @@
 from tests.functional import *
 from string import Template
 import os
-import tempfile
-import xml.etree.ElementTree as ET
+import defusedxml.ElementTree as ET
```

Comment:
This looks good,
    defusedxml.ElementTree - Defused xml.etree.ElementTree facade

Reply:
Thank you @vccardenas